content/js2/blocks/mean/index.md

```diff
@@ -0,0 +1,37 @@
++++
+title = '📊 Calculating the mean'
+headless = true
+time = 15
+facilitation = false
+emoji= '🧩'
+[objectives]
+1='Describe how to compute the mean of a data set in JavaScript'
++++
+
+Let's consider another problem.
+Besides the median, we can also calculate the mean of a set of numbers.
+
+_Given_ an array of numbers,
+_When_ we call `calculateMean` with the array of numbers
+_Then_ we get the mean
+
+Let's create a test to check its functionality:
+
+```js
+test("calculates the mean of a list of numbers", () => {
+  const list = [3, 50, 7];
+  const currentOutput = calculateMean([3, 50, 7]);
+  const targetOutput = 20;
```

Comment:
I was concerned about this - I'm going to leave as it is for now ( because I'd have to go through and update it in other places too! ). However, we can review this again later on if people find the naming is unclear here.